Week1/solution5.py

```diff
@@ -0,0 +1,7 @@
+#Write a python script to enter any string and count vowel.
+n=0
+str1=input("Write a string: ")
+for i in str1:
+    if i=='A' or i=='E' or i=='I' or i=='O' or i=='U' or i=='a' or i=='e' or i=='i' or i=='o' or i=='u':

```

Comment:
any other and i say better way to write this statment

Reply:
yes sir, a list with all vowel characters in both upper case and lower case can be created and to check vowel, membership operator IN can be used.
```
ch = input("Enter a string: ")
n=0
vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']  
for i in ch:
    if i in vowels:
        n=n+1
print("The number of vowels in string: ",n)
```